dumpling: add csv line terminator option (#14844)
pingcap/docs: dumpling-overview.md

| `--csv-separator`            | Separator of each value in CSV files. It is not recommended to use the default ','. It is recommended to use '\|+\|' or other uncommon character combinations| ','                                                                                                                                                                                                                                                                                               | ','                                        |
| `--csv-null-value`           | Representation of null values in CSV files                                                                                                                                                                                                                                                                                         | "\\N"                                      |

> | `--csv-line-terminator`      | The terminator at the end of a line. When exporting data to a CSV file, you can pass in the desired terminator with this parameter. This parameter supports "\r\n" and "\n". Because quotes in bash have different escaping rules, if you want to specify LF as a terminator, you can use a syntax similar to the following: `--csv-line-terminator $'\n'`. | "\r\n" |

Author: ```suggestion
| `--csv-line-terminator`      | The terminator at the end of a line. When exporting data to a CSV file, you can pass in the desired terminator with this parameter. This parameter supports "\r\n" and "\n". The default value is "\r\n", which is consistent with the historical versions. Because quotes in bash have different escaping rules, if you want to specify LF as a terminator, you can use a syntax similar to the following: `--csv-line-terminator $'\n'`. | "\r\n" |
```

> | `--csv-line-terminator`      | The terminator at the end of a line. When exporting data to a CSV file, you can pass in the desired terminator with this parameter. This parameter supports "\r\n" and "\n". The default value is "\r\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF as a terminator, you can use a syntax similar to the following: `--csv-line-terminator $'\n'`. | "\r\n" |

Author: ```suggestion
| `--csv-line-terminator`      | The terminator at the end of a line. When exporting data to a CSV file, you can specify the desired terminator with this parameter. This parameter supports "\r\n" and "\n". The default value is "\r\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to the following: `--csv-line-terminator $'\n'`. | "\r\n" |
```

> | `--csv-line-terminator`      | The terminator at the end of a line. When exporting data to a CSV file, you can specify the desired terminator with this parameter. This parameter supports "\r\n" and "\n". The default value is "\r\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to the following: `--csv-line-terminator $'\n'`. | "\r\n" |

Reviewer: ```suggestion
| `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\r\n" and "\n". The default value is "\r\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\r\n" |
```
